antismash/common/secmet/features/feature.py

```diff
@@ -9,7 +9,10 @@
 from Bio.SeqFeature import SeqFeature, FeatureLocation, CompoundLocation
 from Bio.Seq import Seq
 
-from antismash.common.secmet.locations import convert_protein_position_to_dna
+from antismash.common.secmet.locations import (
+    convert_protein_position_to_dna,
+    location_bridges_origin
```

Comment:
The implementation is only added in the next patch, does it make sense to reorder these?